### PR TITLE
Checkpoints on Steroids

### PR DESCRIPTION
**Pull request by alisterdev**
- [x] Always loads you at your last saved location, at the last saved scene
- [x] Option to disable checkpoints and disable at initial position
- [x] Checkpoint holds light + player drain it when collects 
- [x] Player moves to the next scene upon checkpoint collection
- [x] **BONUS**: no more red cube

### CODE REVIEW

File: ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+EditorBuildSettings:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_Scenes:
+  - enabled: 1
+    path: Assets/Scenes/Level Design/Level Design 1.unity
+  - enabled: 1
+    path: Assets/Scenes/Level Design/Level Design 2 - Volcanoes and Whirlpool.unity
+  - enabled: 1
+    path: Assets/Scenes/Level Design/Level Design 3 - Jellyfish Fields.unity
```
Comment (stellarxo) [inline ProjectSettings/EditorBuildSettings.asset]: Can you include these scenes with the added checkpoints for testing?

Also, do these paths work even though there are spaces?

Comment (stellarxo) [inline ProjectSettings/EditorBuildSettings.asset]: I said for testing, but I meant for the demo today

Comment (alisterdev) [inline ProjectSettings/EditorBuildSettings.asset]: I didn't change scenes or anything, this is just a settings file where you need to indicate all scenes that will be used

If you want, I add this checkpoint to each scene, just afraid of conflicts if someone else (i.e Simon) is working on them

Comment (stellarxo) [inline ProjectSettings/EditorBuildSettings.asset]: he's not working on them

**Pull request merged**